snapcraft/internal/sources/_subversion.py

```diff
@@ -80,14 +80,12 @@ def _get_source_details(self):
         source = self.source
         commit = self.source_commit
 
-        if not commit:  # retrieve the commit id
-            lines = subprocess.check_output(['svn', 'info', self.source_dir]
-                                            ).decode('utf-8').split('\n')
-            prefix = 'Last Changed Rev: '
-            for line in lines:
-                if line.startswith(prefix):
-                    commit = line.replace(prefix, '')
-                    break
+        if not commit:
+            commit = subprocess.check_output(
+                ['svn', 'info',
+                 '--show-item', 'last-changed-revision',
+                 '--no-newline',
```

Comment:
Why is the `--no-newline` parameter needed? 

Maybe it's better (as simpler) without it?

Reply:
@facundobatista since we are already using  `str.strip`, `--no-newline` should not be necessary, however I decided to keep it to make it more explicit.